web-app/CKS.Web.Test/IntegrationTests/ResponseHeaderTests.cs

```diff
@@ -0,0 +1,85 @@
+using CKS.Web.Test.IntegrationTests.Infrastructure;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace CKS.Web.Test.IntegrationTests
+{
+	public class ResponseHeaderTests : TestBase
+	{
+		public ResponseHeaderTests(ITestOutputHelper output) : base(output) {}
+
+		[Theory]
+		[InlineData("test.json")]
+		[InlineData("test.html")]
+		[InlineData("test.xml")]
+		public async void ReturnsMustRevalidateNoCacheHeader(string filename)
+		{
+			// Arrange
```

Comment:
With only 3 lines in these tests they arrange/act/assert is pretty clear so these comments in each test don't add anything - please can you remove?